figures/S5.supplemental_figure5/scripts/S5.r

```diff
@@ -0,0 +1,1448 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2))) # plotting
+suppressPackageStartupMessages(suppressWarnings(library(dplyr))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(patchwork))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(cowplot))) # figure composition
+suppressPackageStartupMessages(suppressWarnings(library(RcppTOML))) # parsing config file
+suppressPackageStartupMessages(suppressWarnings(library(pheatmap))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(lattice))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(gplots))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap))) # heatmap
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify))) # grob
+suppressPackageStartupMessages(suppressWarnings(library(viridis))) # color
+suppressPackageStartupMessages(suppressWarnings(library(platetools))) # make plate plot
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+suppressPackageStartupMessages(suppressWarnings(library(reshape2))) # data manipulation
+suppressPackageStartupMessages(suppressWarnings(library(stringr))) # string manipulation
+suppressPackageStartupMessages(suppressWarnings(library(purrr))) # data manipulation
+source("../../utils/figure_themes.r")
+
+# set the cell type
+cell_type <- "PBMC"
+
+
+# set the path to the data files
+df_stats_path <- file.path(
+    paste0("../../../6.bulk_Morphology_Elastic_Network/2.test_model/results/regression/",cell_type,"/aggregated_with_nomic/model_stats.csv"
+    )
+    )
+df_variance_path <- file.path(
+    paste0("../../../6.bulk_Morphology_Elastic_Network/2.test_model/results/regression/",cell_type,"/aggregated_with_nomic/variance_r2_stats.csv"
+    )
+)
```

Comment:
I am not familiar with R conventions, is newline being used correctly here? It seems strange to have a new line in the middle of a `file.path(` call and not between them.

Reply:
R does not really have a convention like python's PEP so its a free for all. I will make things for consistent though